src/show.jl

```diff
@@ -31,18 +31,25 @@ function Base.show(io::IO, t::AbstractTerm)
     end
 end
 
+lessthanone(x::Real) = x < 0
+lessthanone(x::Complex) = false
```

Comment:
This should mabe be called `lessthanzero` or `isnegative` instead. You can do `isnegative(x) = false` as this will also be useful for other types like JuMP affine expression where `x < 0` would not work.